lib/ddtrace/contrib/active_record/configuration/connection_specification.rb

```diff
@@ -0,0 +1,301 @@
+require 'uri'
+
+# NOTE: This code is copied directly from ActiveRecord.
```

Comment:
WDYT about putting this file in a folder named `vendor`? Similarily how gems are put in `ROOT/vendor` but here it would be something like:

`lib/ddtrace/contrib/active_record/vendor/configuration/connection_specification.rb`

Its a small and cosmetic change, but it would make me feel much better about including files from different projects verbatim.

We could actually insert this file without any modifications too, and add small note in the vendor folder, and exclude the whole prefix in the Rubocop.

Reply:
Hmmm, I like that idea. Would make the Rubocop and other stuff much simpler to deal with too. I'll try this.